src/Cedar/Cedar.c

```diff
@@ -1554,6 +1554,8 @@ CEDAR *NewCedar(X *server_x, K *server_k)
 
 	StrCpy(c->OpenVPNDefaultClientOption, sizeof(c->OpenVPNDefaultClientOption), OVPN_DEF_CLIENT_OPTION_STRING);
 
+	c->OpenVPNPushDummyIPv4AddressOnL2Mode = true; // Default true. Override by the config file.

```

Comment:
Shouldn't this be handled in `SiLoadInitialConfiguration()`?

Reply:
I wrote this in the `NewCedar()` function because similar OpenVPN configuration option `OpenVPNDefaultClientOption` is also written in the `NewCedar()` function.

Reply:
No problem, for consistency we can move both in a dedicated pull request.